db/migrate/20180115214144_add_announcement_to_organization_contents.rb

```diff
@@ -0,0 +1,5 @@
+class AddAnnouncementToOrganizationContents < ActiveRecord::Migration
+  def change
+    add_column :organization_contents, :announcement, :string, default: "", index: true
```

Comment:
why do we need an index on this possibly large string?

Reply:
good point.